app/views/relationships/index.html.haml

```diff
@@ -0,0 +1,21 @@
+%section.people
+  %header
+    %h2 People I Follow
+  %table.table
+    %thead
+      %tr
+        %th(width="30%")
+        %th(width="20%") Videos in Queue
+        %th(width="20%") Followers
+        %th(width="30%") Unfollow
+    %tbody
+      - @relationships.each do |relationship|
+        %tr
+          %td
+            %img(src="http://www.gravatar.com/avatar/#{Digest::MD5.hexdigest(relationship.leader.email.downcase)}?s=40")
```

Comment:
I would recommend to use `= image_tag ...` as its easier to read and I would move gravatar url to a `gravatar_for` helper method, its too long for human eyes, method would be easier to understand + you could reuse it.